geoscilabs/em/TDEMGroundedSource.py

```diff
@@ -85,9 +85,9 @@ def choose_model(model):
 def run_simulation(fname="tdem_gs_half.h5", sigma_block=0.01, sigma_halfspace=0.01):
     from SimPEG.electromagnetics import time_domain as tdem
     from SimPEG.electromagnetics.utils import waveform_utils
-    from SimPEG.EM import Analytics, mu_0
+    from SimPEG.electromagnetics import mu_0
```

Comment:
we should probably just import directly from scipy here 
```suggestion
    from scipy.constants import mu_0
```